### PR TITLE
zh-translate:content/zh/news/releases/1.5.x/announcing-1.5/change-notes/index.md

### PR DESCRIPTION
**Pull request by gauliang**
ref:https://github.com/servicemesher/istio-official-translation/issues/1680

- [x] Docs

### CODE REVIEW

File: content/zh/news/releases/1.5.x/announcing-1.5/change-notes/index.md [hunk 1]
```diff
@@ -1,7 +1,79 @@
 ---
-title: Change Notes
-description: Istio 1.5 release notes.
+title: 变更说明
+description: Istio 1.5 发行版说明
```
Comment (gorda) [inline content/zh/news/releases/1.5.x/announcing-1.5/change-notes/index.md]:
```suggestion
description: Istio 1.5 发行版说明。
```
这会导致无法 lint

File: content/zh/news/releases/1.5.x/announcing-1.5/change-notes/index.md [hunk 2]
```diff
@@ -1,7 +1,79 @@
...
 weight: 10
 ---
 
-TBD
+## 流量管理{#traffic-management}
+
+- **改进** 通过避免不必要的完全推送 [#19305](https://github.com/istio/istio/pull/18164)，提高 `ServiceEntry` 性能。
+- **改进** Envoy sidecar 准备状态探测，可以更加准确地确定准备状态 [#18164](https://github.com/istio/istio/pull/18164)。
```
Comment (gorda) [inline content/zh/news/releases/1.5.x/announcing-1.5/change-notes/index.md]:
```suggestion
- **改进** Envoy sidecar 就绪状态探测，可以更加准确地确定就绪状态 [#18164](https://github.com/istio/istio/pull/18164)。
```

File: content/zh/news/releases/1.5.x/announcing-1.5/change-notes/index.md [hunk 3]
```diff
@@ -1,7 +1,79 @@
...
+- **修复** `kubernetesenv` 适配器可为名字中包含点的 Pod 提供适当支持。
+- **改进** 用于 Mixer 的 Fluentd 适配器，以在导出的时间戳中提供毫秒级支持。
+
+## 配置管理{#configuration-management}
+
+## Operator
+
+- **替换** 将 Alpha `IstioControlPlane` API 替换为新的 [`IstioOperator`](/zh/docs/reference/config/istio.operator.v1alpha1/) API，以便与现有的 `MeshConfig` API 保持一致。
+- **新增** `istioctl operator init` 和 `istioctl operator remove` 命令。
+- **改进** 使用缓存 [`operator#667`](https://github.com/istio/operator/pull/667) 提高对帐速度。
```
Comment (gorda) [inline content/zh/news/releases/1.5.x/announcing-1.5/change-notes/index.md]:
```suggestion
- **改进** 使用缓存 [`operator#667`](https://github.com/istio/operator/pull/667) 提高协调速度。
```

File: content/zh/news/releases/1.5.x/announcing-1.5/change-notes/index.md [hunk 4]
```diff
@@ -1,7 +1,79 @@
...
+
+## `istioctl`
+
+- **毕业** [`Istioctl Analyze`](/zh/docs/ops/diagnostic-tools/istioctl-analyze/)。
+- **新增** 各种分析器：双向 TLS、JWT、`ServiceAssociation`、Secret、sidecar 镜像、端口名称和不建议使用的分析器。
+- **更新** 支持更多针对 `RequestAuthentication` 的验证规则。
+- **新增** [`istioctl analyze`](/zh/docs/ops/diagnostic-tools/istioctl-analyze/) 添加新参数 `-A|--all-namespaces` 以分析整个集群。
+- **新增** [`istioctl analyze`](/zh/docs/ops/diagnostic-tools/istioctl-analyze/) 通过传递 stdin 给的内容进行分析的支持。
```
Comment (gorda) [inline content/zh/news/releases/1.5.x/announcing-1.5/change-notes/index.md]:
```suggestion
- **新增** [`istioctl analyze`](/zh/docs/ops/diagnostic-tools/istioctl-analyze/) 对 stdin 传递的内容进行分析的支持。
```

**Comment by gorda** [comment]
```
- ./public/zh/news/releases/1.5.x/announcing-1.5/change-notes/index.html
  *  linking to /zh/docs/tasks/security/authentication/authn-policy/#auto-mutual-tls, but auto-mutual-tls does not exist (line 11881)
     <a href="/zh/docs/tasks/security/authentication/authn-policy/#auto-mutual-tls">自动双向 TLS</a>
```

@gauliang 缺失一个锚点导致 lint 失败，可以在 `content/zh/docs/tasks/security/authentication/authn-policy/index.md` 第 `82` 行后面加上这部分内容的翻译吗？

```
## Auto mutual TLS

By default, Istio tracks the server workloads migrated to Istio proxies, and configures client proxies to send mutual TLS traffic to those workloads automatically, and to send plain text traffic to workloads without sidecars.

Thus, all traffic between workloads with proxies uses mutual TLS, without you doing
anything. For example, take the response from a request to `httpbin/header`.
When using mutual TLS, the proxy injects the `X-Forwarded-Client-Cert` header to the
upstream request to the backend. That header's presence is evidence that mutual TLS is
used. For example:

{{< text bash >}}
$ kubectl exec $(kubectl get pod -l app=sleep -n foo -o jsonpath={.items..metadata.name}) -c sleep -n foo -- curl http://httpbin.foo:8000/headers -s | grep X-Forwarded-Client-Cert
"X-Forwarded-Client-Cert": "By=spiffe://cluster.local/ns/foo/sa/httpbin;Hash=<redacted>"
{{< /text >}}

When the server doesn't have sidecar, the `X-Forwarded-Client-Cert` header is not there, which implies requests are in plain text.

{{< text bash >}}
$ kubectl exec $(kubectl get pod -l app=sleep -n foo -o jsonpath={.items..metadata.name}) -c sleep -n foo -- curl http://httpbin.legacy:8000/headers -s | grep X-Forwarded-Client-Cert
{{< /text >}}

```

**Comment by gauliang** [comment]
> ```
> - ./public/zh/news/releases/1.5.x/announcing-1.5/change-notes/index.html
>   *  linking to /zh/docs/tasks/security/authentication/authn-policy/#auto-mutual-tls, but auto-mutual-tls does not exist (line 11881)
>      <a href="/zh/docs/tasks/security/authentication/authn-policy/#auto-mutual-tls">自动双向 TLS</a>
> ```
> 
> @gauliang 缺失一个锚点导致 lint 失败，可以在 `content/zh/docs/tasks/security/authentication/authn-policy/index.md` 这个文件的第 `82` 行后面加上这部分内容的翻译吗？
> 
> ```
> ## Auto mutual TLS
> 
> By default, Istio tracks the server workloads migrated to Istio proxies, and configures client proxies to send mutual TLS traffic to those workloads automatically, and to send plain text traffic to workloads without sidecars.
> 
> Thus, all traffic between workloads with proxies uses mutual TLS, without you doing
> anything. For example, take the response from a request to `httpbin/header`.
> When using mutual TLS, the proxy injects the `X-Forwarded-Client-Cert` header to the
> upstream request to the backend. That header's presence is evidence that mutual TLS is
> used. For example:
> 
> {{< text bash >}}
> $ kubectl exec $(kubectl get pod -l app=sleep -n foo -o jsonpath={.items..metadata.name}) -c sleep -n foo -- curl http://httpbin.foo:8000/headers -s | grep X-Forwarded-Client-Cert
> "X-Forwarded-Client-Cert": "By=spiffe://cluster.local/ns/foo/sa/httpbin;Hash=<redacted>"
> {{< /text >}}
> 
> When the server doesn't have sidecar, the `X-Forwarded-Client-Cert` header is not there, which implies requests are in plain text.
> 
> {{< text bash >}}
> $ kubectl exec $(kubectl get pod -l app=sleep -n foo -o jsonpath={.items..metadata.name}) -c sleep -n foo -- curl http://httpbin.legacy:8000/headers -s | grep X-Forwarded-Client-Cert
> {{< /text >}}
> ```



> Auto mutual TLS

这几天没登录，刚看到，现在还需添加吗？

**Pull request merged**